registry/storage/driver/oss/oss.go

```diff
@@ -0,0 +1,827 @@
+// Package oss provides a storagedriver.StorageDriver implementation to
+// store blobs in Aliyun OSS cloud storage.
+//
+// This package leverages the denverdino/aliyungo client library for interfacing with
+// oss.
+//
+// Because OSS is a key, value store the Stat call does not support last modification
+// time for directories (directories are an abstraction for key, value stores)
+//
+// Keep in mind that OSS guarantees only eventual consistency, so do not assume
+// that a successful write will mean immediate access to the data written (although
+// in most regions a new object put has guaranteed read after write). The only true
```

Comment:
AWS has different consistency guarantees in different regions. I was wondering if OSS also has differences between the regions. If not (or if we're not sure), we should remove the part that says "(although in most regions a new object put has guaranteed read after write)". It's a very minor thing - I just want to make sure the comment is accurate.